themes/SuiteP/css/suitep-base/panels.scss

```diff
@@ -1,7 +1,8 @@
 /**** Panels ****/
 
 .panel {
-  margin-bottom: 0;
+  margin-top: 7px;
+  margin-bottom: 0px;
 }
```

Comment:
You can use a shortcut for margin here, like margin: 7 0 0;
Or just skip the 'px' if you use 0.